leveldb/sst/db.go

```diff
@@ -114,13 +137,13 @@ func BuildSSTable(f *os.File, memTable *skiplist.SkipList, tombstones *skiplist.
 			return nil, errors.New("failed to write all bytes")
 		}
 		cumulativeBytesWritten += bytesWritten
-		if cumulativeBytesWritten > sparseIndexThreshold {
+		if cumulativeBytesWritten > ssTableConfig.sparseIndexThreshold {
 			var currentOffset, err = f.Seek(0, io.SeekCurrent)
 			if err != nil {
 				return nil, err
 			}
 			sparseKeys = append(sparseKeys, leveldb.Key(entryToEncode.Key))
-			keyOffsets = append(keyOffsets, offset(currentOffset)) // FIXME: point directly to value
```

Comment:
comment was dumb but there was an issue and this should have been pointing to the start of the key.